docs/rules/no-wait-snapshot.test.md

```diff
@@ -0,0 +1,42 @@
+# Ensures no snapshot is generated inside of a `wait` call' (no-wait-snapshot)
+
+Ensure that no calls to `toMatchSnapshot` are made from within a `wait` method
```

Comment:
Can you make reference to `waitFor` rather than `wait`, and add a note at the end of Rule Details to clarify all util methods this rule reports?

Reply:
Sure thing

Reply:
Do you want to change the name of the rule too? Something like `no-waitfor-snapshot`?

Reply:
That's a good idea! Let's go for `no-wait-for-snapshot` for consistency with `no-wait-for-empty-callback`